packages/react/src/components/Card/CardFooter/CardFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { forwardRef, type HTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import classes from '../Card.module.css';
+
+export type CardFooterProps = {
+  /**
+   * Justifies the direct descendants of the Card Footer
+   * @default spaceBetween
+   * **/
+  justifyContent?: 'flexStart' | 'flexEnd' | 'center' | 'spaceBetween';
+  /**
+   * Alignment of the direct descendants of Card Footer
+   * @default center
+   * **/
+  alignItems?: 'flexStart' | 'flexEnd' | 'center' | 'stretch' | 'baseline';
+  /** Adds a divider to the top of the Card Footer
+   * @default false
+   **/
+  divided?: boolean;
+} & HTMLAttributes<HTMLDivElement>;
+
+export const CardFooter = forwardRef<HTMLDivElement, CardFooterProps>(
+  (
+    {
+      divided = false,
+      justifyContent = 'spaceBetween',
+      alignItems = 'center',
+      className,
+      children,
+      ...rest
+    },
+    ref,
+  ) => (
+    <div
+      {...rest}
+      className={cn(
+        classes.section,
+        classes[`${justifyContent}Justify`],
+        classes[`${alignItems}Align`],
+        { [classes.footerDivided]: divided },
+        className,
+      )}
+      ref={ref}
```

Comment:
I am wondering if it is better to allow the direct css name here, so we don't need to create classes for every possibility?
We might run into some problems with specificity though, maybe we change this once we have a css package @mimarz ?

Reply:
I suggest removing these as there is not point in making an abstraction with what can easily be solved by passing in your own `className`, this is also the way we do it for our other components. 

We can have some padding, gap and flex css properties set by default according to design in `classes.section` or `classes.footer?`, and just have people overwrite whats needed for their use-case with their own `className`.